cmd/tfimport/README.md

```diff
@@ -15,15 +15,23 @@ generated by the Terrarom Engine.
 
 ## Usage
 
+To run against the current working directory:
+
 ```shell
-go install ./cmd/tfimport
-tfimport --input_dir ~/path/to/terraform/configs/dir/
+go install github.com/GoogleCloudPlatform/healthcare-data-protection-suite/cmd/tfimport
+tfimport
```

Comment:
To be clearer you can show two different methods

go install ./cmd/tfimport
cd ~/path/to/terraform/configs/dir
tfimport

go run ./cmd/tfimport --input_dir=~/path...

OR

Put a manfile- like section describing the flags. Then you don't need to provide examples for each flag.